apps/offer-service/src/metrics.ts

```diff
@@ -1,15 +1,24 @@
-import {type CountryPrefix} from '@vexl-next/domain/src/general/CountryPrefix.brand'
+import {Schema} from '@effect/schema'
+import {SqlClient, SqlSchema} from '@effect/sql'
+import {
+  CountryPrefixE,
+  type CountryPrefix,
+} from '@vexl-next/domain/src/general/CountryPrefix.brand'
 import {type OfferId} from '@vexl-next/domain/src/general/offers'
 import {generateUuid} from '@vexl-next/domain/src/utility/Uuid.brand'
 import {MetricsMessage} from '@vexl-next/server-utils/src/metrics/domain'
 import {type MetricsClientService} from '@vexl-next/server-utils/src/metrics/MetricsClientService'
 import {reportMetricForked} from '@vexl-next/server-utils/src/metrics/reportMetricForked'
-import {type Effect} from 'effect'
+import {Array, Effect, Layer, pipe} from 'effect'
 
 const OFFER_PUBLIC_PART_DELETED = 'OFFER_PUBLIC_PART_DELETED' as const
 const OFFER_MODIFIED = 'OFFER_MODIFIED' as const
 const OFFER_CREATED = 'OFFER_CREATED' as const
 const OFFER_REPORTED = 'OFFER_REPORTED' as const
+const TOTAL_BUY_OFFERS = 'TOTAL_BUY_OFFERS' as const
+const TOTAL_SELL_OFFERS = 'TOTAL_SELL_OFFERS' as const
+const TOTAL_SELL_OFFERS_EXPIRED = 'TOTAL_SELL_OFFERS' as const
+const TOTAL_BUY_OFFERS_EXPIRED = 'TOTAL_SELL_OFFERS' as const
```

Comment:
I think values here should be:
`const TOTAL_SELL_OFFERS_EXPIRED = 'TOTAL_SELL_OFFERS_EXPIRED' as const
const TOTAL_BUY_OFFERS_EXPIRED = 'TOTAL_BUY_OFFERS_EXPIRED' as const` ?

Reply:
You are right, thanks for the check!